meta-lmp-bsp/recipes-bsp/trusted-firmware-a/tf-a-fio/0007-feat-mmc-get-boot-partition-size.patch

```diff
@@ -0,0 +1,57 @@
+From 6302a29a75447522c31d43616228055b8f3f2883 Mon Sep 17 00:00:00 2001
+From: Yann Gautier <yann.gautier@st.com>
+Date: Thu, 15 Sep 2022 09:51:21 +0300
+Subject: [PATCH 2/2] feat(mmc): get boot partition size
```

Comment:
Just a comment: interesting that the patchset has the wrong order, the 1st commit uses a function introduced in the 2nd commit :)

Reply:
oops, that was me who messed up :)
in the original MRs it's in correct order